salt/modules/vmctl.py

```diff
@@ -11,9 +11,11 @@
     target machine.
 """
 
+
 import logging
 import re
 
+# Imoprt salt libs:
```

Comment:
Sorry, this header should also be removed

Reply:
removed!!